src/ontology/pato-edit.obo

```diff
@@ -19822,6 +19822,33 @@ is_a: PATO:0000141 ! structure
 created_by: davidos
 creation_date: 2014-12-12T08:43:17Z
 
+[Term]
+id: PATO:0010002
+name: propagation velocity
+def: "The velocity of spatial propagation of a process. Examples include the velocity of propagation of an action potential along an axon." [orcid:0000-0002-7073-9172]
+is_a: PATO:0002062 ! physical quality of a process
+property_value: created_by "dos" xsd:string
+creation_date: 2019-07-18T18:24:15Z
+
+[Term]
+id: PATO:0010003
+name: increased propagation velocity
+def: "A propagation velocity that is higher than normal." [] {http://www.w3.org/2002/07/owl#backwardCompatibleWith="orcid:0000-0002-7073-9172"}
```

Comment:
Orcid notation seems off. Moreover, shouldnt orcids be always full IRIs, rather than short forms?

Reply:
Fixed